react-app/src/components/ResultSidebar.js

```diff
@@ -23,6 +23,15 @@ function ResultSidebar(props) {
       <ResultHeader />
       <div className="sidebar-interior">
         <Accordion defaultActiveKey="0" flush>
+          <Accordion.Item eventKey={-1}>
+            <Accordion.Header
+              onClick={() => {
+                props.setPatientID(-1);
+              }}
+            >
+              Log File
+            </Accordion.Header>
+          </Accordion.Item>
```

Comment:
This is more of a general UI interaction question, but is there anything that will be expanded in the Log File item in this accordion? I know we want to include the resource count for each patient, which will be displayed when the accordion item expands for those items. If there isn't anything similar we want to expand for the log file, does it make sense to move this out of the accordion and have it just be a button/label/some other clickable item above the accordion for the patient results?

Reply:
I put it in the accordion because I wanted it to have the same styling as the rest of the items in the list, and because otherwise, it creates some minor issues with toggling between the log file and patient data. When the log file isn't an Accordion.Item, clicking on it doesn't deactivate the patient data that was selected before, even though it changes the display, so the name of the patient remains highlighted. Additionally, the areas of the ListGroup.Item that are clickable are different from from the areas of the Accordion.Item that are clickable, which may annoy users. 

I have a way to remove the arrow with CSS, so I could do that instead of making it a separate component. That way it can keep all the functionality of the accordion without looking like a dropdown.

Reply:
It turns out that the method for removing the arrow icon removes the arrow icon from every accordion item, because all the arrows are have the CSS style .accordion-button::after. Do you think it's better to keep the arrow, or to make the log file a ListGroup.Item?

Reply:
After talking at stand up, we'll leave the log file in the accordion, and I've created a separate Jira task [here](https://jira.mitre.org/browse/STEAM-631) for including the log statistic summary that we will implement later (after prioritizing it).